src/Interpreter.h

```diff
@@ -74,7 +74,7 @@ class Interpreter {
     using relation_map = std::map<std::string, InterpreterRelation*>;
 
     /** Evaluate value */
-    RamDomain evalVal(const RamExpression& value, const InterpreterContext& ctxt = InterpreterContext());
+    RamDomain evalExpr(const RamExpression& value, const InterpreterContext& ctxt = InterpreterContext());
```

Comment:
One more tiny value -> expression
```suggestion
    RamDomain evalExpr(const RamExpression& expr, const InterpreterContext& ctxt = InterpreterContext());
```

and the matching method in the cpp file